### DISCUSSION TITLE
Migrating Archived spaces

### DISCUSSION THREAD

**Discussion by krizza84** [WikiTransformationProject/wikitraccs-releases#131] [General]
Hi there,

We want to know if there is an option to migrate archived Confluence spaces/pages. We need to retain archived spaces and want to avoid restoring them pre-migration. Thanks

**Comment by heinrich-ulbricht** [comment]
Good question! Let me check that and get back to you.

**Reply by heinrich-ulbricht** [reply]
@krizza84 WikiTraccs does not filter for state when retrieving the list of spaces and populating the Space Inventory. Archived spaces should be listed and can be selected for migration.

Please let me know if there are any issues.

**Comment by krizza84** [comment]
Thank you for your reply Heinrich. Will test this out. Thank you

Kind regards,

Christian Karis
IT Support Officer

P: 0419 411 113
E: ***@***.******@***.***>

exchangeforchange.com.au<https://www.exchangeforchange.com.au/>

***@***.***
